config.default

```diff
@@ -132,6 +132,15 @@ mpasInterpolationMethod = bilinear
 startYear = 1
 endYear = 9999
 
+[index]
+## options related to producing nino index.
+
+# start and end years for the nino 3.4 analysis.  Using out-of-bounds values
+#   like start_year = 1 and end_year = 9999 will be clipped to the valid range
+#   of years, and is a good way of insuring that all values are used.
+startYear = 2
+endYear = 6
```

Comment:
The reason we want to keep all of these is that it's vitally important to test the analysis code constantly, on a number of machines, and using a number of runs we want to support.  We have tried to keep a minimum of relevant options in each that a user might be likely to change.

I use a different workflow myself where I feed 3 config files to `run_analysis.py`.  `config.default` happens automatically.  Then I feed in one of the machine config files, and finally a I pass in a config file that sets the output directory, and any options I'm likely to change from both the defaults and the specific run.  If everyone were doing that, we wouldn't need to include some of these in the machine-specific examples.  But I think my workflow is too complicated to become the default.

As always, we're open to suggestions for how to make things smoother for both users and developers.